src/main/java/uk/gov/justice/probation/courtcaseservice/controller/model/CourtCaseRequest.java

```diff
@@ -85,16 +86,20 @@ public CourtCaseEntity asEntity() {
 
         final List<DefendantEntity> defendants = buildDefendants();
 
-        final CourtCaseEntity entity = CourtCaseEntity.builder()
-                .caseId(caseId)
-                .caseNo(caseNo)
-                .sourceType(SourceType.valueOf(Optional.ofNullable(source).orElse(DEFAULT_SOURCE.name())))
-                .hearings(hearings)
+        final HearingEntity entity = HearingEntity.builder()
+                .courtCase(CourtCaseEntity.builder()
+                    .caseId(caseId)
+                    .caseNo(caseNo)
+                    .sourceType(SourceType.valueOf(Optional.ofNullable(source).orElse(DEFAULT_SOURCE.name())))
+                .build()) 
+                // TODO: Remove. This is a temporary measure to allow the application to continue working whilst we update the data structures adding hearingId
+                .hearingId(caseId)
```

Comment:
There's a couple of these knocking about, until we start actually populating the hearingId we can use caseId and hearingId interchangeably.  In the long term when the feature is complete we will need to have a unique id for pre-existing cases with no hearing so using the caseId makes it apparent where this has happened.